### ISSUE TITLE
Cannot find module "./types"

### ISSUE THREAD

**Issue by art0rz** [Loilo/color-blend#13]
With TypeScript 3.1.3, importing any of the exported color-blend members, e.g.:
```typescript
import {colorBurn} from "color-blend";
```
 results in error:
```
Cannot find module './types'.
```

**Comment by Loilo** [comment]
Thanks for reporting, I'll take a look. 👍

**Comment by Loilo** [comment]
My bad, I forgot to include the `types.ts` in the building step. I just published `color-blend` 2.0.3, the issue should be fixed now.
